layers/meta-resin-intel/recipes-bsp/grub/grub-efi_%.bbappend

```diff
@@ -1,7 +1,11 @@
 FILESEXTRAPATHS_append_intel-corei7-64 := ":${THISDIR}/files"
 
-SRC_URI_append_intel-corei7-64 = " file://grub.cfg"
+SRC_URI_append_intel-corei7-64 = " \
+    file://grub.cfg \
```

Comment:
I would rename this to grub.cfg.external and change the name when we copy it in the final image ("grub.cfg.external:grub.cfg").